src/kagglehub/competition.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from typing import Optional
+
+from kagglehub import registry
+from kagglehub.handle import parse_competition_handle
+from kagglehub.logger import EXTRA_CONSOLE_BLOCK
+
+logger = logging.getLogger(__name__)
+
+
+def competition_download(handle: str, path: Optional[str] = None, *, force_download: Optional[bool] = False) -> str:
```

Comment:
Another alternative would be `competition_dataset_download`. I am ok with either but love for others to chime in. Maybe create a MM emoji pool and tag Jim and a few other folks.

Reply:
did a poll, for the most part, most people who responded don't have strong feelings for one or the other. 
but did mention competition_download would better match with the current kaggle-api naming with the benefit of being shorter.

let's keep as "competition_download"